Asterisk.2013/Asterisk.NET/Manager/Event/TransportDetailEvent.cs

```diff
@@ -0,0 +1,10 @@
+namespace AsterNET.Manager.Event
+{
+	public class TransportDetailEvent : ResponseEvent
+	{
+		public TransportDetailEvent(ManagerConnection source)
+			: base(source)
+		{
+		}
```

Comment:
Could add some properties to this class.
Asterisk documentation has a list of the values.
See: https://wiki.asterisk.org/wiki/display/AST/Asterisk+14+ManagerEvent_TransportDetail